src/pyfx/model/autocomplete/autocomplete_listener.py

```diff
@@ -5,7 +5,10 @@
 from loguru import logger
 from overrides import overrides
 
-from ..common.jsonpath import JSONPathLexer, JSONPathParser, JSONPathListener
+from ..common.jsonpath.JSONPathLexer import JSONPathLexer
+from ..common.jsonpath.JSONPathParser import JSONPathParser
+from ..common.jsonpath.JSONPathListener import JSONPathListener
+
```

Comment:
Nit: Please remove this line, it fails the style check. See https://github.com/cielong/pyfx/pull/135